tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
That seems to have a pretty big impact on the runtime of the tests. Perhaps it's best to keep it as simple as we need it for now - we could just create a new FlxMouse instance manually. There's even already a setter that handles all the details of that since we needed that in flixel-ui at some point.